README.md

```diff
@@ -10,6 +10,21 @@
 Docker Swarm is native clustering for Docker. It turns a pool of Docker hosts
 into a single, virtual host.
 
+## Swarm Disambiguation
+
+**Docker Swarm standalone**: This project. An API proxy system in which a single Docker host fronts several Docker Engines to make them appear as if they are a single Docker Engine.
```

Comment:
@nishanttotla What about this:

**Docker Swarm standalone**: This project. A native clustering for Docker. It turns a pool of Docker hosts into a single, virtual host using an API proxy system.